taxi/forms.py

```diff
@@ -50,3 +50,30 @@ def validate_license_number(
         raise ValidationError("Last 5 characters should be digits")
```

Comment:
Please, update your version with the master's one. Add tests for the searching functionality.